### PR TITLE
Speed up windows builds

### PR DESCRIPTION
**Pull request by m-kuhn**
also comes with some cleanup of the qgis portfile

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3694cf277faec16d97cbcb5277518513f97b4178: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4946-c04245a6790d8de451e66aa2a7999cc86910522a-x86-android.apk)
</details>

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -76,7 +77,7 @@ jobs:
                 -D SENTRY_DSN=${{ secrets.SENTRY_DSN }} \
                 -D SENTRY_ENV="${APP_ENV}" \
                 -D CMAKE_MSVC_RUNTIME_LIBRARY="MultiThreaded\$<\$<CONFIG:Debug>:Debug>" \
-                -D PKG_CONFIG_EXECUTABLE=${CMAKE_BUILD_DIR}/vcpkg_installed/x64-windows/tools/pkgconf/pkgconf.exe \
+                -D PKG_CONFIG_EXECUTABLE=${CMAKE_BUILD_DIR}/vcpkg_installed/x64-windows-static/tools/pkgconf/pkgconf.exe \
```
Comment (nirvn) [inline .github/workflows/windows.yml:80]: @m-kuhn , wondering if it'd be worth using a variable for the x64-windows-static part of this path (as well as for VCPKG_TARGET_TRIPLET and VCPKG_HOST_TRIPLET). I'm worried we'll end up forgetting about it and if one day we change the triplet cmake file name, we'll end up spending time debugging a mysterious failure :)

**Review by nirvn — APPROVED** [review]

File: .github/workflows/windows.yml (hunk 1 above)
Comment (m-kuhn) [reply .github/workflows/windows.yml:80]: If "making a variable" would be very straightforward and maintainable with github actions I would make things considerably more DRY in general. I'm sure if we rename triplets we will find that out ;-)

Comment (nirvn) [reply .github/workflows/windows.yml:80]: @m-kuhn , OK, auto merge enabled.

**Pull request merged**